model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -1102,12 +1112,18 @@ async def execute(
             llm_metadata = (model_endpoint.record.metadata or {}).get("_llm", {})
             inference_framework = llm_metadata["inference_framework"]
 
+            if request.inference_framework_image_tag == "latest":
```

Comment:
since we convert "latest" to a specific image tag in the create route, we just want to fetch that tag right? if you fetch "latest" again here, we might so the  a later version that the model was not created with?

Reply:
ah I meant this scenario
- Dec 1: latest --> v1, user creates model
- Dec 2: latest --> v2
- Dec 3: user gets model, tag on their model would be v1 not v2